insights/parsers/openshift_kube_log.py

```diff
@@ -0,0 +1,57 @@
+"""
+Container logs
+==============
+
+Module for parsing the log files for Openshift kubelet container logs.
+
+
+ControllerManagerLog - file ``/var/log/pods/openshift-kube-controller-manager*/*/*.log``
+----------------------------------------------------------------------------------------
+
+ApiServerLog - file ``/var/log/pods/openshift-kube-apiserver*/*/*.log``
+-----------------------------------------------------------------------
+
+.. note::
+    Please refer to the super-class :class:`insights.core.LogFileOutput`
+"""
+
+from insights.specs import Specs
+from .. import LogFileOutput, parser
```

Comment:
Please use the uniform style for these importings.
```
from insights.core import LogFileOutput, parser
```
Thanks.

Reply:
updated as `from insights import LogFileOutput, parser` instead of `from insights.core import LogFileOutput, parser
`